.github/workflows/release.yml

```
@@ -50,3 +50,65 @@ jobs:
           files: |
             pie.phar
             pie.phar.asc
+      - name: Store PHAR as artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: phar-binary
+          path: pie.phar
+
+  docker-binary-only-image:
+    needs: release-phar
```

With a rebase, this would need to become:

```suggestion
    needs: build-phar
```

---

Applied during interactive rebase.